plugins/git/commands

```diff
@@ -14,6 +14,7 @@ git_build_app_repo() {
   chmod 755 $TMP_WORK_DIR
   unset GIT_DIR GIT_WORK_TREE
   pushd $TMP_WORK_DIR > /dev/null
+  [[ ! -d "$DOKKU_ROOT/$APP" ]] && dokku apps:create $APP
```

Comment:
I'd personally prefer this to be a function call rather than an invocation of a new `dokku` process.